backend/settings.py

```diff
@@ -38,6 +38,16 @@
 
 AUTH_USER_MODEL = 'backbone.CustomUser'
 
+EMAIL_BACKEND = 'django.core.mail.backends.smtp.EmailBackend'
+EMAIL_HOST = 'smtp.sendgrid.net'
+EMAIL_PORT = 587
+EMAIL_USE_TLS = True
+EMAIL_HOST_USER = os.getenv('USERNAME')
+EMAIL_HOST_PASSWORD = os.getenv('PASSWORD')
+DEFAULT_FROM_MAIL = os.getenv('SENDER_MAIL')
+FRONTEND_LINK = os.getenv('FRONTEND_LINK')
+SENDER_NAME = os.getenv('SENDER_NAME')
+WEBSITE = os.getenv('WEBSITE')
```

Comment:
Byłoby mega fajnie, jakbyś dodał nowe env variables do pliku env-example (uprości to prace innym) i gdybyś mógł wrzucić tajne wartości [tutaj](https://github.com/praca-inz-pjjj/secrets), żeby każdy mógł skopiować i uruchomić aplikację lokalnie u siebie.